examples/conversation_sql/react_sql.ipynb

```diff
@@ -0,0 +1,480 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Perform SQL query in natural language\n",
+    "\n",
+    "In this notebook, we provide an example of using React agent to perform sql query in natural lanugage. \n",
```

Comment:
Add a few sentences to introduce what's dail-sql and attach its pdf link

Reply:
Added